emperor/support_files/vendor/js/three.js-plugins/CanvasRenderer.js

```diff
@@ -112,11 +113,18 @@ THREE.CanvasRenderer = function ( parameters ) {
 	_normal = new THREE.Vector3(),
 	_normalViewMatrix = new THREE.Matrix3();
 
+	/* TODO
```

Comment:
These are issues of three.js so I would say no, and to the best of my knowledge these issues do not restrict any of the functionality we offer.